src/components/NewMessage/NewMessagePollEditor.vue

```diff
@@ -58,6 +68,14 @@
 			<NcButton type="tertiary" @click="emit('close')">
 				{{ t('spreed', 'Dismiss') }}
 			</NcButton>
+			<NcActions v-if="supportPollDrafts" force-menu>
+				<NcActionButton v-if="isModerator" :disabled="!isFilled" @click="createPollDraft">
+					<template #icon>
+						<IconFileEdit :size="20" />
+					</template>
+					{{ t('spreed', 'Save as draft') }}
```

Comment:
I might be influenced by github but I expected to find this button in a menu with "Create draft" but not like we have in CallButton (you can press directly "create draft" but for "Save as draft" needs to click on the down arrow button and then click on it).  So basically, make the 3 dot icon a down arrow and attach it to the "create draft". 

There is a discussion about this type of buttons for call button so we can change it later once we agree on using it.

Reply:
Like this?
![image](https://github.com/user-attachments/assets/17764c72-d862-4415-bc2e-14452136f8e8)

Reply:
cc @marcoambrosini

Reply:
I find really nice and intuitive, maybe 3 dot is to be replaced with down arrow

Reply:
I think these two actions are not related, so I wouldn't group them. Taking into account how advanced this feature is, I would just add an unlabelled "save" button to the left of the primary action. We can also remove the dismiss button since we already have the close dialog close button

Reply:
> to the left of the primary action

Thing is, we might want to extend the actions range (+ export to file, for example). But without a 'Dismiss' button three-dot menu should already look better

Reply:
Dropped dismiss button for now and kept three-dot on the left (like in Media settings)